### PR TITLE
Adding CASE project

### PR DESCRIPTION
**Pull request by SebConejo**
Hello there 👋

I think CASE would be an excellent project for Up for Grabs !

Le me know if you have any feedback.

### CODE REVIEW

File: _data/projects/CASE.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+desc: A frameworkless and Open Source tool to build CRUD applications as dashboards or admin panel.
+site: https://case.app
+tags:
+  - appbuilder
+  - typescript
+  - adminpanel
+  - CRUD
+  - frameworkless
+  - opensource
+  -lowcode
```
Comment (shiftkey) [inline _data/projects/CASE.yml]:
```suggestion
  - lowcode
```

File: _data/projects/CASE.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+upforgrabs:
+  name: CASE
+  link: https://github.com/casejs/CASE/issues
```
Comment (shiftkey) [inline _data/projects/CASE.yml]: This `CASE` value should be a label in your project which is used to tag issues that are ideal for new contributors, and the URL should be a link to that list of issues e.g. `https://github.com/casejs/CASE/labels/help%20wanted`

File: _data/projects/CASE.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: CASE
+desc: A frameworkless and Open Source tool to build CRUD applications as dashboards or admin panel.
+site: https://case.app
+tags:
+  - appbuilder
+  - typescript
+  - adminpanel
+  - CRUD
```
Comment (shiftkey) [inline _data/projects/CASE.yml]: All tags need to be lowercase to be consistent

```suggestion
  - crud
```

**Comment by Yirga2023** [comment]
> Hello there 👋
> 
> I think CASE would be an excellent project for Up for Grabs !
> 
> Le me know if you have any feedback. 

_data/projects/CASE.yml

**Comment by SebConejo** [comment]
> > Hello there 👋
> > I think CASE would be an excellent project for Up for Grabs !
> > Le me know if you have any feedback.
> 
> _data/projects/CASE.yml

What do you mean @Yirga2023 ?

**Review by SebConejo — COMMENTED** [review]
Hello @shiftkey,

I have changed the upforgrabs values. Do you think now CASE can be added to up for grabs ?

File: _data/projects/CASE.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  - appbuilder
+  - typescript
+  - adminpanel
+  - crud
+  - frameworkless
+  - opensource
+  - lowcode
+upforgrabs:
+  name: good first issue
+  link: https://github.com/casejs/CASE/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/CASE.yml:14]: At the moment there are no issues assigned to this label, which means it won't be listed in the search results.

Is this something you're able to update (either now or in the future)?

Comment (SebConejo) [reply _data/projects/CASE.yml:14]: @shiftkey Yes, we will add issues assigned to this label tomorrow.

Thank you

**Review by shiftkey — APPROVED** [review]

**Pull request merged**